### PR TITLE
Added readthedocs support

### PR DESCRIPTION
**Pull request by aig787**
Documentation is now hosted at: http://xonsh.readthedocs.org/en/latest/, pointed at my fork for now.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -98,27 +101,32 @@
 # Sphinx are currently 'default' and 'sphinxdoc'.
 #html_theme = 'default'
 #html_theme = 'altered_nature'
-html_theme = 'cloud'
+#html_theme = 'cloud'
```
Comment (scopatz) [inline docs/conf.py]: Please delete this line

**Comment by scopatz** [comment]
Thanks @aig787 - this looks pretty close, just the one minor change!

**Comment by aig787** [comment]
Done. When it all looks okay to you, if you make an account I'll add you as a maintainer and we can point it to the main repo.

**Comment by scopatz** [comment]
Thanks @aig787!  My username is just 'scopatz'.  If you could add @wrywerytwreywery, too, that'd be great.  Thanks!

**Comment by aig787** [comment]
@scopatz alright you're both added!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**